bundles/org.eclipse.passage.loc.features.core/OSGI-INF/org.eclipse.passage.loc.internal.features.core.FeatureDomainRegistry.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" activate="activate" deactivate="deactivate" name="org.eclipse.passage.loc.internal.features.core.FeatureDomainRegistry">
```

Comment:
Not clear why "activate" was removed

Reply:
It has been performed automatically on rebuild for all OSGiComponents under 2024-03 IDE. This one sink into commit occasionally. Thanks for noticing!